pkg/antctl/transform/networkpolicy/transform_test.go

```diff
@@ -0,0 +1,329 @@
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "c",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:              "b",
+							UID:               "aaa",
+							CreationTimestamp: timeB,
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "b",
+						},
+					},
+				},
+			},
+			opts: map[string]string{
+				"sort-by": "creationTimestamp",
+			},
+			sortedNPList: &cpv1beta.NetworkPolicyList{
+				Items: []cpv1beta.NetworkPolicy{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:              "a",
+							UID:               "abc",
+							CreationTimestamp: timeA,
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "a",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:              "b",
+							UID:               "aaa",
+							CreationTimestamp: timeB,
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "b",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:              "c",
+							UID:               "abb",
+							CreationTimestamp: timeC,
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "c",
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "sort by name default",
+			npList: &cpv1beta.NetworkPolicyList{
+				Items: []cpv1beta.NetworkPolicy{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "a",
+							UID:  "abc",
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "a",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "c",
+							UID:  "abb",
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "c",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "b",
+							UID:  "aaa",
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "b",
+						},
+					},
+				},
+			},
+			opts: map[string]string{
+				"sort-by": "",
+			},
+			sortedNPList: &cpv1beta.NetworkPolicyList{
+				Items: []cpv1beta.NetworkPolicy{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "a",
+							UID:  "abc",
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "a",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "b",
+							UID:  "aaa",
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "b",
+						},
+					},
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "c",
+							UID:  "abb",
+						},
+						SourceRef: &cpv1beta.NetworkPolicyReference{
+							Name: "c",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			result, err := listTransform(tt.npList, tt.opts)
+			sortedList := make([]Response, 0, len(tt.npList.Items))
+			for i := range tt.sortedNPList.Items {
+				o := Response{&tt.sortedNPList.Items[i]}
+				sortedList = append(sortedList, o)
+			}
```

Comment:
I think you can define `expectedResponse` in the `tests` structure to replace `sortedNPList`, then you don't have to process this again in the test function.